resolwe/flow/execution_engines/bash/__init__.py

```diff
@@ -35,7 +35,7 @@ def evaluate(self, data):
             # Include special 'proc' variable in the context.
             inputs['proc'] = {
                 'data_id': data.id,
-                'data_dir': settings.FLOW_EXECUTOR['DATA_DIR'],
+                'data_dir': self.manager.get_executor().resolve_data_path(data, '..'),
```

Comment:
Maybe add dedicated function for this instead of passing `..` as filename?

Reply:
@kostko make the `filename` argument optional and return root data dir if it is not defined.

Reply:
Done.